CHANGELOG.md

```diff
@@ -0,0 +1,25 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/)
+and this project adheres to [Semantic Versioning](http://semver.org/).
+
+## Unreleased
+
+<!-- New PRs should document their changes here. -->
```

Comment:
add a "do not delete this line!" or equivalent so that people don't remove after adding the first unreleased change